src/angularjs/src/app/places/list/places-list-map-directive.js

```diff
@@ -0,0 +1,80 @@
+(function() {
+
+    /* @ngInject */
+    function PlacesListMapController(MapConfig, Neighborhood) {
```

Comment:
I expect they'll diverge more as we add features. Was thinking it might be worth revisiting to see about a refactor once we have more of idea of what else we'll be doing with these (interactivity and popup building, in particular.)